about.html

```diff
@@ -88,7 +90,7 @@ <h5 class="company-title fw-bolder small pt-2">Kenya <br> Creatives Department (
 
     <section class="mt-4 container px-3 past-exhibitions">
       <div class="title d-flex flex-column justify-content-center">
-        <h3 class="text-dark h4 fw-bold mt-4  text-center">See the past art exhibitions</h3>
+        <h3 class="text-dark h3 fw-bold mt-4  text-center">See the past art exhibitions</h3>
         <hr class="hr mx-auto border border-1 border-danger">
 
         <p class="email-contact mx-md-auto text-center p-2 mb-3 text-opacity-75">
```

Comment:
- [ ]  Currently, your entire `about page` looks jumbled up It doesn't look as it did when you submitted it the first time. You might have messed up with the styles when you were trying to make the requested changes so kindly ensure that the page looks properly styled as it did before and ensure that you fix all the requested changes the previous reviewer asked on the `about page` for both desktop and mobile :100: 
- Your Design Now
[Screencast from 03-15-2023 09:35:21 PM.webm](https://user-images.githubusercontent.com/102699958/225450635-df894dd7-d91b-482f-98b5-8e993327deba.webm)

- The Original Design
![image](https://user-images.githubusercontent.com/102699958/225450563-3b247f3f-596d-428b-99b1-eed7d3949831.png)